app/src/main/java/org/p2p/wallet/striga/signup/interactor/StrigaSignupInteractor.kt

```diff
@@ -156,6 +156,10 @@ class StrigaSignupInteractor(
             }
         }
 
+        // not critical, we can ignore errors from this call
+        kotlin.runCatching {
+            userInteractor.loadAndSaveUserStatusData().unwrap()
```

Comment:
you can create a method ignore or just call userInteractor.loadAndSaveUserStatusData()

until you call unwrap - no exception will be thrown